### ISSUE TITLE
Add ability to skip options with default values for UnParserSettings

### ISSUE THREAD

**Issue by MichaelMankiewicz** [commandlineparser/commandline#541]
I have many command line parameters with default values in most of them. My app generates `.bat` file to run itself with parameters which passed to it's main instance. Now `Parser.Default.FormatCommandLine` makes long string with all parameters, most of which has default values and can be omitted.
I want to be able to set `UnParserSettings` to skip parameters that has default values when I use it as 
`FormatCommandLine` action's parameter. Is it possible?

**Comment by moh-hassan** [comment]
I agree with you to skip the parameters that have default values.
This enhancement  may be available in the next release.

**Comment by moh-hassan** [comment]
I just want to confirm this test case:

```cs
class Options
        {
            [Option(Default = 99)] //should be skipped when P1=99
            public  int P1 { get;set;}
            [Option()]
            public  string P2 { get;set;}
            [Option(Default = 88)] //should be skipped when P3=88
            public  int P3 { get;set;}
        }

var options = new Options{P2="xyz" ,P1=99, P3=88}	;

  args should be:  --p2 xyz   
```
Is it right?

**Comment by MichaelMankiewicz** [comment]
Yes, this is exactly what I mean

**Comment by moh-hassan** [comment]
@MichaelMankiewicz 
The issue is fixed and you can try the nuget Package downloaded from CI [server](https://ci.appveyor.com/api/buildjobs/iy7242cxnac19owf/artifacts/src%2FCommandLine%2Fbin%2FRelease%2FCommandLineParser.2.7.0-alpha-48-hgkdqqrq.nupkg)
I appreciate you if you can feedback your test result to merge the PR.

**Comment by moh-hassan** [comment]
Example:
```cs
  var options = new Options_With_Defaults {P2 = "xyz", P1 = 99, P3 = 88,P4= Shapes.Square }  ;
       var args=     new Parser()
              //Configure Unparser: SkipDefault=true
                .FormatCommandLine(options,x=>x.SkipDefault=true) ;

    class Options_With_Defaults
    {
        [Option(Default = 99)]
        public int P1 { get; set; }
        [Option()]
        public string P2 { get; set; }
        [Option(Default = 88)]
        public int P3 { get; set; }
        [Option(Default = Shapes.Square)]
        public Shapes P4 { get; set; }   //enum option
    }

```
result:
```
 args= --p2 xyz
```

**Comment by moh-hassan** [comment]
Fixed in v2.7+, See [wiki with online demo](https://github.com/commandlineparser/commandline/wiki/Unparsing#skipping-options-with-default-values)
